### PR TITLE
Fix/clear-interactions

### PR DESCRIPTION
**Pull request by g-francesca**
Ref ORM-1846

- Add slot to optionally add content for empty state
- Fixes autoFocus textearea on mount
- Reset interactions on unmount

### CODE REVIEW

File: apps/storybook/stories/public/orama-chat-box.stories.tsx [hunk 1]
```diff
@@ -72,7 +72,11 @@ const Template = ({ preset }) => {
       sourceBaseUrl=${preset?.sourceBaseUrl}
       .sourcesMap=${preset?.sourcesMap}
       .suggestions=${preset?.suggestions}
-    ></orama-chat-box>
+    >
+    <div slot="empty-state" style="display: flex; justify-content: center; align-items: center; padding: 1rem;">
+      <orama-text as="p">Optional empty content here...</orama-text>
+    </div>
```
Comment (faustoq) [inline apps/storybook/stories/public/orama-chat-box.stories.tsx:78]: can you please remove it from the orama-chat-box story? the "empty-state" slot usage will be documented in https://docs.orama.com if needed.

File: packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.tsx [hunk 2]
```diff
@@ -53,7 +54,12 @@ export class ChatBox {
           sourceBaseUrl={this.sourceBaseUrl}
           sourcesMap={this.sourcesMap}
           suggestions={this.suggestions}
-        />
+          focusInput={this.autoFocus}
+        >
+          <div slot="empty-state">
+            <slot name="empty-state" />
+          </div>
```
Comment (faustoq) [inline packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.tsx:61]:
```suggestion
          <slot name="empty-state" />
```

File: apps/storybook/stories/public/orama-chat-box.stories.tsx (hunk 1 above)
Comment (g-francesca) [reply apps/storybook/stories/public/orama-chat-box.stories.tsx]: removed

File: packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.tsx (hunk 2 above)
Comment (g-francesca) [reply packages/ui-stencil/src/components/orama-chat-box/orama-chat-box.tsx:61]: I tried this way but it's not working, as we need to place the custom empty state content within the `orama-chat `child component within the dedicated slot area (see `orama-chat`)
I initially did something like:
`<slot name="empty-state" slot="chat-empty-state"/>` 
It was actually working but seems to be not a standard orthodox approach, so I removed it :')

**Review by faustoq — APPROVED** [review]

**Pull request merged**